<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(130, 212, 157);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_00">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>50</y>
      <width>95</width>
      <height>78</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(189, 238, 205 );
font: 700 48pt &quot;Script MT Bold&quot;;
border-radius:  25%;
min-width: 56px;
min-height: 56px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_01">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>96</width>
      <height>78</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(189, 238, 205 );
font: 700 48pt &quot;Script MT Bold&quot;;
border-radius:  25%;
min-width: 56px;
min-height: 56px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="NewGame_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>19</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15%;
background-color:rgb(189, 238, 205 );</string>
    </property>
    <property name="text">
     <string>New Game</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="twoplayer_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>player vs player</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="playerwithpc_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>133</width>
      <height>27</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>player vs cpu </string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>9</y>
      <width>84</width>
      <height>35</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="about_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>19</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:  15%;
background-color:rgb(86, 190, 203);</string>
    </property>
    <property name="text">
     <string>about</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_02">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>50</y>
      <width>95</width>
      <height>78</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(189, 238, 205 );
font: 700 48pt &quot;Script MT Bold&quot;;
border-radius:  25%;
min-width: 56px;
min-height: 56px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_11">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>134</y>
      <width>96</width>
      <height>78</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(189, 238, 205 );
font: 700 48pt &quot;Script MT Bold&quot;;
border-radius:  25%;
min-width: 56px;
min-height: 56px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_10">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>134</y>
      <width>95</width>
      <height>78</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(189, 238, 205 );
font: 700 48pt &quot;Script MT Bold&quot;;
border-radius:  25%;
min-width: 56px;
min-height: 56px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_12">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>134</y>
      <width>95</width>
      <height>78</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(189, 238, 205 );
font: 700 48pt &quot;Script MT Bold&quot;;
border-radius:  25%;
min-width: 56px;
min-height: 56px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_21">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>218</y>
      <width>96</width>
      <height>78</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(189, 238, 205 );
font: 700 48pt &quot;Script MT Bold&quot;;
border-radius:  25%;
min-width: 56px;
min-height: 56px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_20">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>218</y>
      <width>95</width>
      <height>78</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(189, 238, 205 );
font: 700 48pt &quot;Script MT Bold&quot;;
border-radius:  25%;
min-width: 56px;
min-height: 56px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_22">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>218</y>
      <width>95</width>
      <height>78</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>48</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(189, 238, 205 );
font: 700 48pt &quot;Script MT Bold&quot;;
border-radius:  25%;
min-width: 56px;
min-height: 56px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="showscoreO">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:  15%;
font: 700 15pt &quot;Script MT Bold&quot;;
background-color:rgb(189, 238, 205 );
color:rgb(85, 137, 102);</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="showscoreX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(182, 235, 241);
border-radius:  15%;
font: 700 15pt &quot;Script MT Bold&quot;;
background-color:rgb(86, 190, 203);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="signO">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>24</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>O</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="signX">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>24</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>315</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
